test/phoenix_pubsub/tracker_state_test.exs

```diff
@@ -0,0 +1,116 @@
+defmodule Phoenix.TrackerStateTest do
+  use ExUnit.Case
```

Comment:
We'll have to revisit this, but I made it false because we are spawning trackers on nodes. If we keep the registered tracker names and topics unique it may be okay however